core/src/main/scala/stryker4s/config/Config.scala

```diff
@@ -12,7 +12,7 @@ case class Config(files: Seq[String] = Seq("**/main/scala/**/*.scala"),
                   baseDir: File = File.currentWorkingDirectory,
                   testRunner: TestRunner = CommandRunner("sbt", "test"),
                   reporters: List[MutantRunReporter] = List(new ConsoleReporter),
-                  logLevel: Level = Level.INFO) {
+                  logLevel: Level = Level.DEBUG) {
```

Comment:
With the logging level being set right after it's read. Should we still have the default debug? I think a default of info makes more sense

Reply:
The reason why I made the default DEBUG is because of config reader. In that function we are printing the default config that has been used: `debug("Config used: " + defaultConf.toHoconString)` if we set the default to INFO this log statement will never be able to print. 

We could make the default INFO but then I would like to have the config print info as well.

What would you suggest?

Reply:
well the thing you're missing this, it's all happing inside `tryRecoverFromFailures` which only goes off when there is no `stryker4s.conf` available. So it will purely rely on the defaults set in the config class. Which would be loglevel INFO and thus never print.

Reply:
Ah. Yeah I missed that haha. In that case we could change the log to info. Or maybe we should move the used config to always be logged (at debug) and just accept that it won't be logged in that case.

Reply:
For now i changed it to INFO and also logged the default config to INFO. I liked that idea the best.